src/TicketingSystem.DataAccess/Entities/PriceOption.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace TicketingSystem.DataAccess.Entities
+{
+    public class PriceOption : BaseEntity
+    {
+        public string Name { get; set; }
+
+        public decimal Coefficient { get; set; }
```

Comment:
I'm in opinion that this document is redundant - we can have a PriceOption enum instead which value will be assigned to all EventSeats (or on the Event/Section/Row level). And then in the BLL, we would have a specific price calculation logic for each option.

Reply:
Removed the class, added related enum